src/sentry/tasks/reports.py

```diff
@@ -393,21 +393,53 @@ def build_message(timestamp, duration, organization, user, report):
     return message
 
 
+DISABLED_ORGANIZATIONS_USER_OPTION_KEY = 'reports:disabled-organizations'
+
+
+def user_subscribed_to_organization_reports(user, organization):
+    return organization.id not in set(
```

Comment:
I'm pretty sure a linear check would be faster here, since the cost of converting to a `set()` would be linear anyways.